debezium-connector-sqlserver/pom.xml

```diff
@@ -20,7 +20,7 @@
         <sqlserver.user>sa</sqlserver.user>
         <sqlserver.password>Password!</sqlserver.password>
         <sqlserver.dbname>testDB</sqlserver.dbname>
-        <docker.filter>microsoft/mssql-server-linux:2017-CU9-GDR2</docker.filter>
+        <docker.filter>mcr.microsoft.com/mssql/server:2019-GA-ubuntu-16.04</docker.filter>
```

Comment:
I have to update SQLServer image because with 2017 one the SQLServer Agent was stopped just after container is started I don't know why

Reply:
Interesting. Should be ok to change it.